### PR TITLE
Fix pasting multi-paragraph plain text chunks when cleanPaste is enabled

### PR DESCRIPTION
**Pull request by yacomink**
Also includes a stop-gap for #601

### CODE REVIEW

File: src/js/paste.js [hunk 1]
```diff
@@ -124,9 +125,10 @@ var PasteHandler;
 
                 if (!pastedHTML) {
                     pastedHTML = pastedPlain;
+                    onlyText = true;
```
Comment (j0k3r) [inline src/js/paste.js]: What if you didn't introduce this new variable `onlyText` and just removed this `if`?
On the next if below, `pastedHTML` will be empty and the condition will be false.

**Comment by nmielnik** [comment]
Will #603 fix your issue, and thus you don't need the stop-gap?

**Comment by nmielnik** [comment]
So I've checked in a fix for the issue you opened, and cut a new release.  Sorry to change things underneath you, but you'll need to update your fork with the new changes

**Comment by yacomink** [comment]
good lookin out Nick, your update fixed my issue. I'm going to close this one and submit a rebased one

**Comment by nmielnik** [comment]
ok, apologies again for the extra overhead, but I really wanted to get the issue you and @j0k3r found fixed and released as quickly as possible

**Pull request closed without merging**